modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -143,5 +129,33 @@ def self.get_page_configuration(page, position)
         config[page] = { type: :text, position: }
       end
     end
+
+    def generate_prawn_document(stamp, page_configuration)
+      stamp_path = Rails.root.join(Common::FileHelpers.random_file_path)
+      Prawn::Document.generate(stamp_path, margin: [0, 0]) do |pdf|
+        page_configuration.each do |config|
+          case config[:type]
+          when :text
+            pdf.draw_text stamp[:text], at: config[:position], size: stamp[:font_size] || 16
+          when :new_page
+            pdf.start_new_page
+          end
+        end
+      end
+    end
+
+    def call_datestamp_pdf(stamped_template_path, desired_stamp, append_to_stamp)
+      Rails.logger.info('Calling PDFUtilities::DatestampPdf', current_file_path:, stamped_template_path:)
+      datestamp_instance = PDFUtilities::DatestampPdf.new(stamped_template_path, append_to_stamp:)
+      coords = desired_stamp[:coords]
+      datestamp_instance.run(text: desired_stamp[:text], x: coords[0], y: coords[1], text_only: true, size: 9)
+    end
+
+    def get_auth_text_stamp
+      current_time = "#{Time.current.in_time_zone('America/Chicago').strftime('%H:%M:%S')} "
```

Comment:
Why is the timezone set to Chicago?

Reply:
@ericboehs Good question. [Looks like back in May '23 we determined that Central Mail was using Central Time.](https://github.com/department-of-veterans-affairs/vets-api/pull/12667)